AnnoyancesFilter/sections/popups.txt

```diff
@@ -119,6 +119,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+mackolik.com##.rup-sticky
```

Comment:
This is AD leftover.
It seems added by third-party script `cdn.eksiup.com/api/special/rupmackolik.com.arsiv.js`
`||cdn.eksiup.com/api/special/rup$script`
https://publicwww.com/websites/%22cdn.eksiup.com%2Fapi%2Fspecial%2F%22+depth%3Aall/